vendor/github.com/containers/storage/drivers/chown.go

```diff
@@ -154,6 +154,11 @@ func NewNaiveLayerIDMapUpdater(driver ProtoDriver) LayerIDMapUpdater {
 // UIDs and GIDs for "host" values from the second map which correspond to the
 // same "container" IDs.
 func (n *naiveLayerIDMapUpdater) UpdateLayerIDMap(id string, toContainer, toHost *idtools.IDMappings, mountLabel string) error {
+
+	if os.Getuid() != 0 {
```

Comment:
It seems to me that this decision should happen at some higher layer (which will _understand_ that the change has not actually taken place) instead of silently doing nothing and subverting expectations.  Up to @nalind , though.

Reply:
yes I agree, I've added this patch only to enable this use case but it is definitely not the final way on how this should be handled in containers/storage